File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int(" enter a number = ");

    }
    while (n < 1 || n >8);
        
    for (int i = 0; i < n; i++)
    {
        for (int a =(i + 1); a < n; a++)
        {
            printf(" ");
        }


        for (int j = 0; j <= i; j++)
        {
            printf("#");

        }
            
        printf("\n");
    }       
       
}          

File: week 6/mario.py
from cs50 import get_int

# Prompts user for height of the half-pyramid
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

# Prints the half-pyramid
for i in range(height):
    for j in range(height - i - 1):
        print(" ", end="")
    for j in range(i + 1):
        print("#", end="")
    print()